app/uk/gov/hmrc/fhregistrationfrontend/forms/deregistration/DeregistrationReason.scala

```diff
@@ -14,6 +14,22 @@
  * limitations under the License.
  */
 
+/*
```

Comment:
this to be removed. It seems it got double license here. Maybe delete both, recompile and should add just one